components/atom/input/src/Input/Features/Addon/index.scss

```diff
@@ -4,6 +4,13 @@ $bgc-atom-input-addon: $c-gray-lightest !default;
 $bdc-atom-input-addon: $c-gray-light !default;
 $c-atom-input-addon: $c-gray-light !default;
 
+$l-atom-input-icon: $gutter !default;
+$w-atom-input-icon: $sz-icon-m !default;
+$h-atom-input-icon: $sz-icon-m !default;
+$t-atom-input-icon: 50% !default;
```

Comment:
50% and -50% is the same value.
But they don't need to have variables anyway.

Reply:
Is the same vale but they refer to different properties so I think it's better to have different variables for each value. Also these values refer to Icon positioning that may be related to Icon dimensiones that are configurable, so I think these value should also be configurable